pkg/hub/manager.go

```diff
@@ -78,9 +79,15 @@ func (o *AddOnControllerOptions) RunControllerManager(ctx context.Context, contr
 
 	kubeInformer := informers.NewSharedInformerFactoryWithOptions(kubeClient, 5*time.Minute, informers.WithNamespace(o.Namespace))
 
+	service, err := kubeClient.CoreV1().Services(config.HUB_NAMESPACE).Get(ctx, config.ANP_SERVICE_NAME, metav1.GetOptions{})
```

Comment:
if we  can use service name, we don't need this

Reply:
Sure